app/bulk_resource/views.py

```diff
@@ -1,8 +1,11 @@
 import csv
+from datetime import datetime
 from flask import redirect, render_template, url_for
 from flask.ext.login import login_required
 from . import bulk_resource
```

Comment:
nit: move ".", "..", and others start start with a symbol to before the first one that starts with a letter